cloudify_cli/tests/commands/test_secrets.py

```diff
@@ -89,3 +89,26 @@ def test_secrets_create_mutually_exclusive_arguments(self):
             exception=SystemExit
         )
         self.assertIn('mutually exclusive with arguments:', outcome.output)
+
+    def test_secrets_export_password_length(self):
+        self.invoke('cfy secrets export -p 1234567',
+                    err_str_segment='ERROR: Password must contain at least 8 '
+                                    'characters',
+                    exception=CloudifyValidationError)
+
+    def test_secrets_export_password_empty(self):
+        self.invoke('cfy secrets export -p ' '',
+                    err_str_segment='2',
+                    exception=SystemExit)
+
+    def test_secrets_export_mutually_exclusive_tenant_all_tenants(self):
+        self.invoke('cfy secrets export -t default_tenant -a',
+                    err_str_segment='2',
+                    exception=SystemExit)
```

Comment:
Make sure the error is as you expect self.assertIn('`tenant_name` is mutually exclusive with arguments: [all_tenants]', outcome.output)